eva-lib/src/main/java/uk/ac/ebi/eva/lib/repository/VariantEntityRepository.java

```diff
@@ -60,4 +63,11 @@ Long countByChromosomeAndStartAndReferenceAndAlternate(String chromosome, int st
 
     @Query(value = "{'chr': ?0, 'start': ?1, 'ref': ?2}", count = true)
     Long countByChromosomeAndStartAndReference(String chr, int start, String ref);
+
+    @Query(value = "{'chr': ?0, 'start': ?1, 'alt': ?2, 'files.sid': {$in : ?3}}}", count = true)
+    Long countByChromosomeAndStartAndEndAndAltAndStudyIn(String chr, int start, String alt, List<String> studyIds);
+
+    @Query(value = "{'chr': ?0, 'start': ?1, 'type': ?2, 'files.sid': {$in : ?3}}}", count = true)
```

Comment:
I'm not sure if this is possible, to somehow inject strings into query annotations. When we first started on the Spring repositories for variants, and we wanted to query start and end values where start was both less than the specified end, but also greater than the specified start minus a certain margin (e.g. `db.variants_1_2.find({ "chr" : 10 , "end"  : { $gt :  5000000, $lt : 6000000  }, "start"  : { $lt :  6000000, $gt : 4000000 }})` for a querying a variant at position 5000000).
I think this was one of the reasons the repository had to be manually implemented (in addition to needing to apply the filters).